app/services/container_topology_service.rb

```diff
@@ -114,15 +114,12 @@ def retrieve_providers
   end
 
   def build_kinds
-    kinds =  { :Replicator => true,
-               :Pod        => true,
-               :Container  => true,
-               :Node       => true,
-               :Service    => true,
-               :Host       => true,
-               :VM         => true,
-               :Route      => true
-             }
+    base_kinds = [:Replicator, :Pod, :Container, :Node, :Service, :Host, :VM, :Route]
+    kinds = {}
+    base_kinds.each do |kind|
+      kinds[kind] = true
+    end
```

Comment:
zip is interesting :)
I'm refactoring the other part of the method to reduce copy paste as well (deeper than @matthewd suggestion)